packages/contract/lib/reason.js

```diff
@@ -23,14 +29,28 @@ const reason = {
       res && typeof res === "object" && typeof res.result === "string";
 
     if (isObject) {
```

Comment:
Does this condition -- which is perhaps named a little misleadingly -- still apply in the Ganache 7 case?